linux_os/guide/system/auditing/auditd_configure_rules/directory_ownership_var_log_audit/tests/correct_value.pass.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+chgrp root /var/log/audit
```

Comment:
Shouldn't there be `chown` instead because the rule is about user ownership not group ownership?

Reply:
That's correct.